react/Header/Header.less

```diff
@@ -33,6 +27,7 @@
   align-items: center;
   flex: 1 0 auto;
   padding: 0 (@grid-gutter-width / 2);
+  max-width: @grid-container-width + @grid-gutter-width;
```

Comment:
The 10px (half gutter) existed from all the way back when the Header lived in Chalice added here https://github.com/SEEK-Jobs/chalice/commit/a9817bc72a49ca74efd787bd4e3005138660b30b I guess only @michaeltaranto would know thats if he remembered...